### DISCUSSION TITLE
Pich to Zoom for SkiaImage

### DISCUSSION THREAD

**Discussion by Agredo** [taublast/DrawnUi#140] [Q&A] [answered]
I am searching for .Net MAUI a simple way to make pinch to zoom possible. 

Does have DrawnUI a way to zoom in pictures with pinch to zoom (best case would be mouse wheel too)?

**Comment by taublast** [comment, marked as answer]
Yes please check the demo app, mouse wheel is supported too: https://github.com/taublast/DrawnUi.Maui.Demo/blob/main/src/AllDrawn/Views/Content/PopupGallerySlider.xaml
